relay/kusama/src/lib.rs

```diff
@@ -1504,6 +1521,8 @@ impl pallet_nis::Config for Runtime {
 	type MaxIntakeWeight = MaxIntakeWeight;
 	type ThawThrottle = ThawThrottle;
 	type RuntimeHoldReason = RuntimeHoldReason;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkSetup = ();
```

Comment:
The NIS pallet only requires a dedicated `BenchmarkSetup` when we use the `Assets` pallet for the `Counterpart` currency (to force create the asset) but since we use a `Balances` pallet here, it should work without.